lib/active_merchant/billing/gateways/payu_latam.rb

```diff
@@ -192,15 +193,15 @@ def add_buyer(post, payment_method, options)
           buyer[:dniType] = buyer_hash[:dni_type]
           buyer[:cnpj] = buyer_hash[:cnpj] if options[:payment_country] == 'BR'
           buyer[:emailAddress] = buyer_hash[:email]
-          buyer[:contactPhone] = options[:shipping_address][:phone] if options[:shipping_address]
+          buyer[:contactPhone] = (options[:billing_address][:phone] if options[:billing_address]) || (options[:shipping_address][:phone] if options[:shipping_address])
           buyer[:shippingAddress] = shipping_address_fields(options) if options[:shipping_address]
         else
           buyer[:fullName] = payment_method.name.strip
           buyer[:dniNumber] = options[:dni_number]
           buyer[:dniType] = options[:dni_type]
           buyer[:cnpj] = options[:cnpj] if options[:payment_country] == 'BR'
           buyer[:emailAddress] = options[:email]
-          buyer[:contactPhone] = (options[:shipping_address][:phone] if options[:shipping_address]) || (options[:billing_address][:phone] if options[:billing_address])
+          buyer[:contactPhone] = (options[:billing_address][:phone] if options[:billing_address]) || (options[:shipping_address][:phone] if options[:shipping_address])
```

Comment:
Ditto. If you intend to create `:contacTPhone` unconditionally and are okay with it being `nil`, this is fine, but I don't think that's what you intended.